backend/server.js

```diff
@@ -43,7 +43,6 @@ app.use('/googlegroups', groupRouter)
 
 app.listen(port, async () => {
     console.log(`Server is running on port: ${port}`)
-    await updateGroup()
```

Comment:
what is the purpose of removing this

Reply:
I was the one who originally added it while testing, honestly it makes sense to keep it so im fine with leaving it there